sam.c

```diff
@@ -1646,7 +1646,7 @@ static bool cmd_write(Vis *vis, Win *win, Command *cmd, const char *argv[], Sele
 		if (!path)
 			return false;
 
-		struct stat meta;
+		struct stat meta = { 0 };
 		bool existing_file = !stat(path, &meta);
 		bool same_file = existing_file && file->name &&
 		                 file->stat.st_dev == meta.st_dev && file->stat.st_ino == meta.st_ino;
```

Comment:
I don't think this is necessary because:

- if stat(2) fails, it returns -1
- existing_file is !-1 which is 0
- same_file is 0 && [...] which is 0

but meta is only accessed if these variables are true.

Reply:
It is not necessary, but I was confused by strange numbers while debugging. Feel free to ignore.